### DISCUSSION TITLE
Docs: Invalid download link in 'Installation' section

### DISCUSSION THREAD

**Discussion by stenno** [semaphoreui/semaphore#2044] [Q&A]
### Problem

At https://docs.semui.co/administration-guide/installation#package-manager , the docs suggest a link:
wget https://github.com/semaphoreui/semaphore/releases/\
download/v2.9.58/semaphore_2.9.44_linux_amd64.deb

However, this link results in a 404. A valid link to the latest release is currently:
https://github.com/semaphoreui/semaphore/releases/download/v2.9.75/semaphore_2.9.75_linux_amd64.deb

**Comment by tboerger** [comment]
Please open an issue or a fixing PR at the docs repository https://github.com/semaphoreui/semaphore-docs
